src/dashboard/category-results-screen.js

```diff
@@ -15,19 +16,73 @@ function Paper ({ paper }) {
   ]);
 }
 
+function findRelativeDate (papers, range) {
+  const now = new Date();
+  const today = now.toISOString().split('T')[0];
+  const daysAgo = sub(now, { days: 4 }).toISOString();
+  const weekAgo = sub(now, { weeks: 1 }).toISOString();
+  const monthAgo = sub(now, { months: 1 }).toISOString();
+  const displayPapers = [];
+
+  for (const paper of papers) {
+    const paperDate = paper.date.toISOString().split('T')[0];
+
+    if (range === 'today' && paperDate === today) { // Today's papers
+      displayPapers.push(paper);
+    } else if (range === 'days' && paperDate >= daysAgo && paperDate < today) { // Papers from 4 days ago
```

Comment:
I attempted to use isWithinInterval here, but soon realized it had a lot of complications with timezones - the downloaded data would be in UTC, but it would get automatically converted to EST when comparing it to the current date which made things a bit more difficult to work with. I found that it was easier to manipulate date strings rather than Date objects for our purposes.